### PR TITLE
Added Repo Icon in Context Box

### PR DESCRIPTION
**Pull request by JapneetSingh5**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description
Added Repo Icon In Context Box.

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Screen Shot 2020-06-23 at 9 26 11 PM
Screen Shot 2020-06-23 at 9 42 38 PM

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by JapneetSingh5** [comment]
I'm having problems making a PR with only the new commits, how do I resolve that? Is it because I haven't updated my fork with the main repo?

**Comment by bdougie** [comment]
does you fork have the latest changes from master?

I might suggest the following. 

```
git remote upstream set-url  https://github.com/open-sauced/open-sauced
git fetch
git rebase upstream master 
```

**Comment by JapneetSingh5** [comment]
> does you fork have the latest changes from master?
> 
> I might suggest the following.
> 
> ```
> git remote upstream set-url  https://github.com/open-sauced/open-sauced
> git fetch
> git rebase upstream master 
> ```
> 
> Let me know if this helps or something you are already doing.

Didn't work, getting the same '9 commits ahead of master' even after doing this. Should probably have worked in a different branch for this.
Is this PR mergeable or not?

File: src/styles/RepositoryAvatar.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+import {borderRadius} from "./variables";
+
```
Comment (yg) [inline src/styles/RepositoryAvatar.js:3]:
```suggestion
import styled from "styled-components";
import {borderRadius, margin, size} from "./variables";
```

Import this to use `gutter` and other sizes

File: src/styles/RepositoryAvatar.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const RepositoryAvatar = styled.img`
+  margin-right: 10px !important;
+  margin-bottom: -10px;
```
Comment (yg) [inline src/styles/RepositoryAvatar.js:6]:
```suggestion
  margin-right: ${gutter.gutter} !important;
  margin-bottom: -${gutter.gutter};
```

Use `${gutter.gutter}`

File: src/styles/RepositoryAvatar.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  display:inline;
+  object-fit: cover;
+  height: 40px;
+  width: 40px;
```
Comment (yg) [inline src/styles/RepositoryAvatar.js:10]:
```suggestion
  height: ${gutter.medium};
  width: ${gutter.medium};
```

Use `${gutter.medium}`

File: src/styles/RepositoryAvatar.js [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  -moz-border-radius: 5px;
+  -webkit-border-radius: ${borderRadius};
+  border-radius: 5px;
```
Comment (yg) [inline src/styles/RepositoryAvatar.js:13]:
```suggestion
  -moz-border-radius: ${borderRadius};
  -webkit-border-radius: ${borderRadius};
  border-radius: ${borderRadius};
```

Use `${borderRadius}`

File: src/styles/RepositoryAvatar.js [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+const RepositoryAvatar = styled.img`
+  margin-right: 10px !important;
+  margin-bottom: -10px;
+  display:inline;
+  object-fit: cover;
+  height: 40px;
+  width: 40px;
+  -moz-border-radius: 5px;
+  -webkit-border-radius: ${borderRadius};
```
Comment (yg) [inline src/styles/RepositoryAvatar.js:12]: Do we need this?

Comment (JapneetSingh5) [reply src/styles/RepositoryAvatar.js]: I kept that as other components (eg Avatar.js) also had prefixing for cross-browser compatibility, to maintain consistency.

File: src/styles/RepositoryAvatar.js (hunk 4 above)
Comment (JapneetSingh5) [reply src/styles/RepositoryAvatar.js]: Done.

File: src/styles/RepositoryAvatar.js (hunk 3 above)
Comment (JapneetSingh5) [reply src/styles/RepositoryAvatar.js]: Done.

File: src/styles/RepositoryAvatar.js (hunk 2 above)
Comment (JapneetSingh5) [reply src/styles/RepositoryAvatar.js]: Done.

File: src/styles/RepositoryAvatar.js (hunk 1 above)
Comment (JapneetSingh5) [reply src/styles/RepositoryAvatar.js]: Done.

**Comment by bdougie** [comment]
@JapneetSingh5 the extra commits are fine because we squash the commits when merging.

**Comment by JapneetSingh5** [comment]
> @JapneetSingh5 the extra commits are fine because we squash the commits when merging. 

oh, then thats fine
but I will still take care from next time onwards to update the fork before I make changes, thanks again! ☺️

**Pull request merged**